README.md

```diff
@@ -3,22 +3,18 @@
 
 Spits out some stats about webpack compilation process to a file.
 
-<br>
-
 ## Install
 
 ```bash
 npm install --save-dev webpack-bundle-tracker
 ```
 
-<br>
-
 ## Compatibility
 
 This project is compatible with NodeJS versions 12 and up.
 
-
-<br>
+## Migrating from version 1.x.y to 2.x.y
+Starting on version 2.0.0, when creating a new instance of `BundleTracker`, the usage of the `path` parameter has been fixed and it's now being used to generate the output path for the stats file, together with the `filename` parameter. On version 2.0.0, if the `path` parameter is ommited from the constuctor call, it will use its default value and may end up placing the stats file inside an incorrect directory. To avoid that, when migrating, double-check if the file placement is as expected. The usage of these parameters is documented [here](#usage) and [here](#options).
```

Comment:
"it will use its default value and may end up placing the stats file inside an incorrect directory." clarify please what the default is: where it places the file.

Reply:
Updated this one and the table on the footer to clarify the behavior (that it'll use the `output.path` before trying to use root directory).